### ISSUE TITLE
hard evt.preventDefault() in Drag handlers

### ISSUE THREAD

**Issue by lolenko** [luster-io/impulse#36]
https://github.com/luster-io/impulse/blob/master/lib/drag.js#L79
because of it i can't make only horizontal slider with live vertical page scroll.

My suggest - completely remove this lines from Drag _start, _move, _end methods. And leave it to the discretion of users.
Is not that  why Drag instance generates events like move?

I'm really sorry for my english.

**Comment by xcoderzach** [comment]
I think the reason for that was that chrome on android only fires touchmove once, unless evt.preventDefault is called.  I'm not sure if that's still the case.

**Comment by lolenko** [comment]
Yes, it is. But what can i do if i don't need to prevent browser native scroll(exactly this case i can't solve now) or default behavior of nested link or another control? If I need i have access to original event via drag.on('start', callback).

**Comment by lolenko** [comment]
Put links in this(https://github.com/luster-io/demo-bounce-scroll) example and you will find what i mean on touch device? The links will not work! I agree it's not right.

**Comment by lolenko** [comment]
If you prevent touchstart you also prevent all mouse events including click!
